### PR TITLE
car park schema updated using schema.org

### PR DESCRIPTION
**Pull request by fersagui**

### CODE REVIEW

File: car-park.swagger.yaml [hunk 1]
```diff
@@ -90,31 +67,56 @@ definitions:
...
-  TimePeriod:
-    description: Information about time periods
+  OpeningHours:
     type: string
-    example:
-      - "24/7" # all week
-      - "Mo-Fr" # all day from Mondays to Fridays
-      - "08:00-21:00" # all week from 8:00 to 21:00
-      - "Mo-Fr 06:30-21:00; Sa 08:00-14:00"  # a very descriptive example
+    description: Opening times
```
[the remark this answers is not in the document]
Comment (fersagui) [reply car-park.swagger.yaml]: Ok, sure. I will adjust previous examples in our actual schema.

File: car-park.swagger.yaml [hunk 2]
```diff
@@ -154,18 +156,18 @@ definitions:
...
-        - type: time
-          maxDurationInMinutes: 120
+          price: 20,
+          type: time
+        - maxDurationInMinutes: 120
           price: 32
         - type: additional
           maxDurationInMinutes: 60
-          price: 10
+          price: 10,
```
[the remark this answers is not in the document]
Comment (fersagui) [reply car-park.swagger.yaml]: My mistake. I have listed the price objects in the wrong way. I will fix them.

File: car-park.swagger.yaml [hunk 3]
```diff
@@ -90,31 +67,62 @@ definitions:
...
+     url: "http://schema.org/openingHours"
     example:
-      - "24/7" # all week
+      - "Mo-Su" # all week
       - "Mo-Fr" # all day from Mondays to Fridays
-      - "08:00-21:00" # all week from 8:00 to 21:00
-      - "Mo-Fr 06:30-21:00; Sa 08:00-14:00"  # a very descriptive example
+      - "Mo-Su 08:00-21:00" # all week from 8:00 to 21:00
+      - "Tu,Th 08:00-12:00" # Tuesdays and Thursdays from 8:00 to 12:00
+      - ["Mo-Fr 06:30-21:00", "Sa 08:00-14:00"]  # using an array for a complex one
```
Comment (nighthawk) [inline car-park.swagger.yaml]: The last example doesn't match the schema. Can you update the schema accordingly? What we'd want is that `OpeningHours` uses `oneOf`, which is then either a string or an array of strings. Need to switch to Open API 3.0 for that though. See https://openapi-converter.herokuapp.com

Comment (nighthawk) [reply car-park.swagger.yaml]: Though that's poor API design. How about we just make this an array of `OpeningHours` strings instead wherever it's being used? And usually it's just a single string in the array? Makes the API simpler to use and parse.

File: car-park.swagger.yaml [hunk 4]
```diff
@@ -90,31 +67,56 @@ definitions:
...
+      telephone:
         type: string
-      website:
+      logo:
         type: string
-        format: url
-      color:
-        $ref: '#/definitions/Color'
+      image:
+        type: string
```
[the remark this answers is not in the document]
Comment (nighthawk) [reply car-park.swagger.yaml]: For images we could use an array of `http://schema.org/ImageObject`, where we primarily use `contentUrl`, `width`, `height` and `fileFormat`. And the consumer of the API can then pick the image of the format and size best suited to them.

File: car-park.swagger.yaml (hunk 3 above)
Comment (fersagui) [reply car-park.swagger.yaml]: > How about we just make this an array of `OpeningHours` strings instead wherever it's being used? 
Yes. I would prefer this. Much easier to use and parse.

File: car-park.swagger.yaml (hunk 4 above)
[the remark this answers is not in the document]
Comment (nighthawk) [reply car-park.swagger.yaml]: See https://github.com/skedgo/maas-apis/pull/1/commits/2b73af215168a946b6d64f8feff7a2b034b95aab

File: car-park.swagger.yaml [hunk 5]
```diff
@@ -90,31 +67,56 @@ definitions:
...
+      #  - postalcode
+      #  - elevation
+    required:
+      - latitude
+      - longitude
+
+  Provider:
+    # based on: 'http://schema.org/provider'
+    externalDocs:
+      url: 'https://schema.org/Organization'
```
[the remark this answers is not in the document]
Comment (nighthawk) [reply car-park.swagger.yaml]: See https://github.com/skedgo/maas-apis/pull/1/commits/b40130d08a85abbcbfb69fd03601ed8215a4c520

**Comment by nighthawk** [comment]
@fersagui please align with what I did here: https://github.com/skedgo/maas-apis/pull/1/commits/b40130d08a85abbcbfb69fd03601ed8215a4c520

**Comment by fersagui** [comment]
> @fersagui please align with what I did here: skedgo/maas-apis@b40130d

**Comment by nighthawk** [comment]
@fersagui, yes, that's fine. I would treat it like this: What's in maas-api is the minimum recommended set, what's in FIWARE is the set of all the standardised fields, what we use in this repo is what the TripGo routing engine supports:
 
tripgo - 2018-06-16 10 34 00

**Pull request merged**